packages/frontend/pages/index.tsx

```diff
@@ -2,16 +2,38 @@ import React from "react";
 import { graphql } from "react-relay";
 import { QueryWrapper } from "@wdp/lib/api/components";
 import { useRouteSlug } from "@wdp/lib/routes";
+import { fetchQuery } from "relay-runtime";
+import { environment } from "@wdp/lib/app";
 import InstanceContentLayout from "components/composed/instance/InstanceContentLayout";
 import { pagesQuery as Query } from "@/relay/pagesQuery.graphql";
-
+import {
+  pagesIndexQuery,
+  pagesIndexQueryResponse,
+} from "@/relay/pagesIndexQuery.graphql";
 import AppLayout from "components/global/AppLayout";
+import { AppHtmlHead } from "components/global";
+
+export async function getStaticProps() {
+  const env = environment();
```

Comment:
It seems strange to me that I have to make a new environment here. Since this data is on the frontend and would not require a sign in, I'm skipping the keycloak token.